src/main/java/org/jenkinsci/plugins/electricflow/Configuration.java

```diff
@@ -91,7 +91,7 @@ public boolean getIgnoreSslConnectionErrors()
         return this.ignoreSslConnectionErrors;
     }
 
-    public String getElectricFlowPassword()
+    public Secret getElectricFlowPassword()
```

Comment:
would you consider to keep the former method, annotating with `@Deprecated` and create a new one returning the `Secret`? If we don't care about breaking binary compatibility here, we can just remove the old constructor too.

Reply:
Let me check that. But you are probably correct.

Reply:
In fact, no. The code cannot have two methods, with the same name, same parameter (no parameter), even if it's returning a different type.

Reply:
Yes, as we've discussed, I was suggesting to have a new method, with a different name, obviously, maybe: `getElectricFlowSecret()` and deprecate the old one, the same way you did with the constructor. To keep the binary compatibility. But as we've discussed, if you think it isn't worth it because the plugin is not consumed externally, I'm fine with it.